packages/nc-gui-v2/components/cell/MultiSelect.vue

```diff
@@ -99,6 +143,28 @@ watch(
 </template>
 
 <style scoped>
+.ms-close-icon {
+  color: rgba(0, 0, 0, 0.25);
+  cursor: pointer;
+  display: flex;
+  font-size: 12px;
+  font-style: normal;
+  height: 12px;
+  line-height: 1;
+  text-align: center;
+  text-transform: none;
+  transition: color 0.3s ease, opacity 0.15s ease;
+  width: 12px;
+  z-index: 1;
+  margin-right: -6px;
+  margin-left: 3px;
```

Comment:
I'm okay with regular CSS but I'd rather see tailwind used instead, to keep it more consistent.

As an example:
```css
.ms-close-icon {
	@apply cursor-pointer text-sm leading-4 text-center;
	@apply flex w-[12px] h-[12px] z-1 ml-[3px] mr-[-6px];
	@apply before:block hover:text-white/45;
}
```

some stuff might not be possible of course, you can fall back to regular css there but I won't force you to change it 😄